src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/TagletWriterImpl.java

```diff
@@ -479,7 +474,6 @@ private Content linkSeeReferenceOutput(Element holder,
                                            DocTree refTree,
                                            String refSignature,
                                            Element ref,
```

Comment:
A related but different improvement would be to change this from `Element ref` to `TypeMirror refType`.